Kuroba/app/src/main/java/com/github/adamantcheese/chan/features/gesture_editor/Android10GesturesExclusionZonesHolder.kt

```diff
@@ -0,0 +1,236 @@
+package com.github.adamantcheese.chan.features.gesture_editor
+
+import android.graphics.Rect
+import com.github.adamantcheese.chan.core.settings.ChanSettings
+import com.github.adamantcheese.chan.core.settings.ChanSettings.EMPTY_JSON
+import com.github.adamantcheese.chan.utils.AndroidUtils.isAndroid10
+import com.github.adamantcheese.chan.utils.Logger
+import com.google.gson.Gson
+
+class Android10GesturesExclusionZonesHolder(
+        private val gson: Gson,
+        private val minScreenSize: Int,
```

Comment:
Rather than passing these into the constructor, just get them as needed.

Reply:
Done.